testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -487,6 +492,7 @@
             <properties>
                 <app.server>wildfly</app.server> <!--in case the profile is called directly-->
                 <app.server.skip.unpack>false</app.server.skip.unpack>
+                <arquillian.protocol.default>Servlet 5.0</arquillian.protocol.default>
```

Comment:
I am wondering if we still need to handle this in a profile, it seems that the base property is set to `Servlet 3.0`, but perhaps we can raise the minimum version here as well?